api/wallet.go

```diff
@@ -364,6 +397,30 @@ func (api *API) walletSiafundsHandler(w http.ResponseWriter, req *http.Request,
 	})
 }
 
+// walletSweepHandler handles API calls to /wallet/sweep.
+func (api *API) walletSweepHandler(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+	// Get the seed using the ditionary + phrase
+	dictID := mnemonics.DictionaryID(req.FormValue("dictionary"))
+	if dictID == "" {
+		dictID = "english"
+	}
+	seed, err := modules.StringToSeed(req.FormValue("seed"), dictID)
+	if err != nil {
+		WriteError(w, Error{"error when calling /wallet/sweep: " + err.Error()}, http.StatusBadRequest)
+		return
+	}
+
+	coins, funds, err := api.wallet.SweepSeed(seed)
+	if err != nil {
+		WriteError(w, Error{"error when calling /wallet/sweep: " + modules.ErrBadEncryptionKey.Error()}, http.StatusBadRequest)
+		return
+	}
+	WriteJSON(w, WalletSweepPOST{
+		Coins: coins,
+		Funds: funds,
+	})
+}
```

Comment:
If I understand correctly, there is currently no support for sweeping a key (such as a siag key)?

Do we want to have those under the same endpoints, or do we want to specify them using two separate endpoints? `/wallet/sweep/seed` and `/wallet/sweep/siag` and `/wallet/sweep/033x`?

We should be able to sweep all types of keys that we support loading, which I believe is the above 3.